[Bug] Unable to resolve dependency tree. peer astro@"^3.2.0 || ^4.0.0" from astro-cloudinary@1.3.0

# **Bug Report**

## **Describe the bug**

After upgrading to Astro 5 (5.0.2), when trying to install astro-cloudinary
```
npm install astro-cloudinary
```
I get the following dependency errors
```
npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: mnchasersite@0.0.1
npm error Found: astro@5.0.2
npm error node_modules/astro
npm error   astro@"^5.0.2" from the root project
npm error
npm error Could not resolve dependency:
npm error peer astro@"^3.2.0 || ^4.0.0" from astro-cloudinary@1.3.0
npm error node_modules/astro-cloudinary
npm error   astro-cloudinary@"1.3.0" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.

```

## **Is this a regression?**

Not sure. However, I thought Cloudinary was a launch partner of Content Layer API - which is an Astro 5 feature?  Confused why ^3.2.0 and ^4.0.0 are even listed in package.json, and ^5.0.0 isn't the only version supported. 
(Edit: Reading old Astro blog posts, I see Content Layer API was released as experimental in Astro@4.14.0, so that makes sense).

## **Steps To Reproduce the error**

1. Install Astro@latest (5.0.2)
2. npm install astro-cloudinary


## **Expected behaviour**

I believe there's a missing dependency for astro@5.0.0  in v1.3.0 package.json.  
```
package.json
------------
...
 "peerDependencies": {
    "astro": "^5.0.0"   <-- missing
...
```


## **CodeSandbox or Live Example of Bug**

## **Screenshot or Video Recording**

### **Your environment**

- OS: MacOS
- Astro Version: 5.0.2
- Node version: 22.9.0
- Npm version: 22.9.0
- Browser name and version: Any


## **Additional context**

--- a/astro-cloudinary/package.json
+++ b/astro-cloudinary/package.json
@@ -47,7 +47,7 @@
     "unpic": "^3.18.0"
   },
   "peerDependencies": {
-    "astro": "^3.2.0 || ^4.0.0"
+    "astro": "^3.2.0 || ^4.0.0 || ^5.0.0"
   },
   "devDependencies": {
     "astro": "^4.15.6"

